jujugui/static/gui/src/app/utils/environment-change-set.js

```diff
@@ -185,31 +164,19 @@ YUI.add('environment-change-set', function(Y) {
       @param {Object} record The individual record object from the changeSet.
     */
     _wrapCallback: function(record) {
-      var args = record.command.args;
-      var index = args.length - 1;
-      var callback;
-      // If the dev doesn't provide a callback when the env call is made we
-      // need to supply one and increment the index that the wrapper gets
-      // placed in. The wrapper must execute as it's required to keep the
-      // changeset in sync.
-      if (typeof args[index] === 'function') {
-        callback = args[index];
-      } else {
-        callback = function() {};
-        index += 1;
-      }
-      // Possible strict violation.
-      var self = this;
-      /**
-        Wrapper function for the supplied callback for the command.
-        @method _callbackWrapper
-      */
-      function _callbackWrapper() {
+      const args = record.command.args;
+      // By convention, the callback is the last argument.
+      const callback = args[args.length - 1];
```

Comment:
This assumes that the user provided the callback - we should probably check that this is indeed a function.